configs/.env

```diff
@@ -0,0 +1,19 @@
+# GITHUB_ACCESS_TOKEN=
```

Comment:
These are only used in tests though. If that doesn't change anything, how would you use these though? 

Only three dependencies are `joi`, `yaml`, and `bluebird`